importer/Monument.py

```diff
@@ -179,17 +191,18 @@ def set_coords(self, coord_keywords_tuple):
             if self.lat == 0 and self.lon == 0:
                 return
             else:
-                ref = self.wlm_source
                 self.add_statement(
-                    "coordinates", (getattr(self, lat), getattr(self, lon)), refs=[ref])
+                    "coordinates", (getattr(self, lat), getattr(self, lon)))
 
     def set_image(self, image_keyword="image"):
         if self.has_non_empty_attribute(image_keyword):
-            self.add_statement("image", getattr(self, image_keyword))
+            self.add_statement("image", getattr(self, image_keyword),
```

Comment:
I would probably sett `image = getattr(self, image_keyword)` on its own line and then use that in `add_statement` to make it easier to read. (same in a few other places)